index.src.html

```diff
@@ -980,6 +983,10 @@ <h4 id="fetch-api">Fetch API</h4>
         |request|'s [=request/client=] is a [=secure context=], then switch on
         |init|["{{RequestInit/targetAddressSpace}}"]:
         <dl class=switch>
+          <dt>public
+          <dd>Set |request|'s [=target IP address space=] to [=IP address
+            space/public=]
```

Comment:
I think we should throw a TypeError here if we already know the value isn't valid.

I'm still not sure that we really need `public` in the `IPAddressSpace` type since we don't have a valid case to use it.  I guess that's why I called it `RequestTargetAddressSpace` in the first place so that we only need to include the valid values that should be used.

```suggestion
          <dd>[=Throw=] a {{TypeError}}.
```

Reply:
I think we need this because we need to define if a request's IP address space is less public then the initiator's. In this case, we need to define initiator's IP address space first, which can be public.

Reply:
We have already defined the public address space in https://wicg.github.io/private-network-access/#ip-address-space-public.  I meant we didn't need it in the IDL, i.e. the JS API.

Reply:
I still think throwing a TypeError here is better than a network error later in the algorithm.

Reply:
I meant we should share the same enum other than create a new one for fetch API only.

I don't think this is a TypeError on one hand, and on the other, "public" is forbidden to bypass mixed content check sounds like a better logic than a fetch request cannot be "public" targetAddressSpace.